languages/indonesian/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply plugin: 'com.android.application'
+
+ext.shouldBePublished = false
+
+dependencies {
+    implementation project(path: ':languages:english:pack')
```

Comment:
should point to the new indonesian pack:
```
implementation project(path: ':languages:indonesian:pack')
```

Reply:
Oh, I missed this. It didn't mentioned in new steps. Another time I  should use silversearcher to find all string that need to be changed.